packages/vision/src/index.js

```diff
@@ -1281,6 +1281,14 @@ Vision.findImages_ = function(images, callback) {
   images = arrify(images);
 
   function findImage(image, callback) {
+    if (Buffer.isBuffer(image)) {
+      process.nextTick(function() {
```

Comment:
@calibr I think we could probably safely drop the `process.nextTick` part off, WDYT?